keyboards/santoku/keymaps/tyetye/keymap.c

```diff
@@ -99,10 +99,6 @@ static bool in_alttab = false; // does an ALT-TAB, for windows cycling, without
 
 const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
 	[_QWERTY] = { /*QWERTY*/
```

Comment:
As long as we're removing unneeded comments. I'd say these can be removed too. I can't make the suggestion on the other lines from this PR interface right now. So I'll let you go remove those. I've removed them in the default keymap.
```suggestion
	[_QWERTY] = { 
```